src/sentry/utils/performance_issues/performance_detection.py

```diff
@@ -706,7 +706,16 @@ def is_creation_allowed_for_project(self, project: Project) -> bool:
 
     @classmethod
     def is_event_eligible(cls, event, project: Project = None) -> bool:
+        request = event.get("request", None) or None
+
         sdk_name = get_sdk_name(event) or ""
+
+        if request:
+            url = request.get("url", "") or ""
+            method = request.get("method", "") or ""
+            if url.endswith("/graphql") or method.lower() in ["post", "get"]:
```

Comment:
🤔 why `["post", "get"]`? Are the other methods okay?

Reply:
According to graphql spec, only POST and GET are used for graphql endpoints
https://graphql.org/learn/serving-over-http/#http-methods-headers-and-body

Reply:
Ah okay roger! So, hold on, I'm little confused, wont `or method.lower() in ["post", "get"]:` ignore basically all requests, then?

Reply:
Ughhh, your right, it should be `and`! I'll update and add a test for that

Reply:
Very nice catch @gggritso !

Reply:
:george_salute:

Reply:
the or has been converted to an and.
and it ensured that non graphql post requests pass as well.